__tests__/app.test.js

```diff
@@ -61,6 +61,44 @@ describe("app", () => {
     });
   });
  });
+ describe("GET /api/articles/:article_id", () => {
+  test("200: responds with a status of 200 when the article is found", () => {
+    const testArticleId = 4;
+    return request(app)
+    .get(`/api/articles/${testArticleId}`)
+    .expect(200);
+  });
+  test("404: responds with a status of 404 when the article is not found", () => {
+    const testArticleId = 40000;
+    return request(app)
+    .get(`/api/articles/${testArticleId}`)
+    .expect(404);
+  });
+  test("400: responds with a status of 400 when the article id is an invalid number", () => {
+    const testArticleId = "Hello";
+    return request(app)
+    .get(`/api/articles/${testArticleId}`)
+    .expect(400);
+  });
+  test("200: respond with an aticle object with the following properties when the article is found", () => {
```

Comment:
Just a typo in this test description - it isn't a big deal but it's good to correct them if you see them as employers will be looking at this portfolio piece 🙂